README.md

```diff
@@ -110,16 +101,11 @@ nakadiClient.stream(eventName)
         .listen(SalesOrderPlaced.class, listener);
 ```
 
-### Fahrschein-JDBC Schema migration
-
-Fahrschein-JDBC provides Flyway schema migrations in the `fahrschein-db` resource folder. You can
-point Flyway to it using the [`resources`](https://flywaydb.org/documentation/configuration/parameters/locations) configuration parameter.
-
 ## Using multiple partitions and multiple consumers
 
 With the `PartitionManager` api it is possible to coordinate between multiple nodes of one application, so that only one node is consuming events from a partition at the same time.
 
-Partitions are locked by one node for a certain time. This requires that every node has an unique name or other identifier.
+Partitions are locked by one node for a certain time. This requires that every node has a unique name or other identifier.
```

Comment:
As you already made a grammar fix, here's another one:
`This requires that every node has` -> `This requires that every node have`

Reply:
sure about that one?

Reply:
maybe "This requires every node to have a unique name or other identifier."?

Reply:
I'm sure about my suggestion, but what you wrote also works.